spec.html

```diff
@@ -38272,8 +38272,9 @@ <h1>synchronizes-with</h1>
           <p>For each pair (_R_, _W_) in _execution_.[[ReadsFrom]], (_W_, _R_) is in _execution_.[[SynchronizesWith]] if all the following are true.</p>
           <ul>
             <li>_R_.[[Order]] is `"SeqCst"`.</li>
-            <li>If _W_.[[Order]] is `"SeqCst"` and _R_ and _W_ have equal ranges.</li>
-            <li>If _W_.[[Order]] is `"Init"`, and for each event _V_ such that (_R_, _V_) is in _execution_.[[ReadsFrom]], _V_.[[Order]] is `"Init"`.</li>
+            <li>_W_.[[Order]] is `"SeqCst"` or `"Init"`.</li>
+            <li>If _W_.[[Order]] is `"SeqCst"` then _R_ and _W_ have equal ranges.</li>
```

Comment:
Existing nit: please add a , before the 'then' for consistency.

Reply:
done